action.yml

```
@@ -21,6 +21,10 @@ inputs:
     description: 'Node.js version to use. If not provided, will use the version specified in the `.nvmrc` file.'
     required: false
     default: ''
+  skip-allow-scripts:
+    description: 'Skip the `yarn allow-scripts` step. If your job does not need this step, skipping it will save 10-15 seconds.'
```

The actual time it saves is very dependant on which scripts are run 😅 

```suggestion
    description: 'Skip the `yarn allow-scripts` step. If your job does not need this step, skipping it may save some time.'
```

---

Yeah I was thinking about the Extension when I wrote that 🙂